Arrangement_on_surface_2/test/Arrangement_on_surface_2/CMakeLists.txt

```diff
@@ -7,7 +7,6 @@ project(Arrangement_on_surface_2_Tests)
 enable_testing()
 
 find_package(CGAL REQUIRED COMPONENTS Core)
-
 include(${CGAL_USE_FILE})
 
 if(COMMAND target_compile_options)
```

Comment:
`target_compile_options` is in CMake since... well at least since CMake 3.0. So that `if/else` is completely useless.

Reply:
It is the file `cgal_test.cmake` that you should modify.